include/seqan3/core/parallel/execution.hpp

```diff
@@ -1,102 +0,0 @@
-// -----------------------------------------------------------------------------------------------------
```

Comment:
Can you please leave the header in, but mark it as deprecated? I.e. add the doxgen deprecation stuff and the `SEQAN3_DEPRECATED_HEADER` but do not delete the content.